UnityProject/Assets/Scripts/Core/Lighting/LightSource.cs

```diff
@@ -253,6 +253,7 @@ private void SetSprites()
 			RefreshBoxCollider();
 		}
 
+		[ClientRpc]
 		private void SetAnimation()
```

Comment:
isn't this called by other stuff?

Reply:
Yes, and it should always sync when getting called.

Reply:
but the problem is then it Wouldn't be called on server?

Reply:
ClientRpc calls on everything, including the server. And it should so values can be synced properly

Reply:
But this won't run on a self hosted build?

Reply:
it will run on a self hosted build